packages/react-sdk/src/component-generator.test.ts

```diff
@@ -687,6 +691,7 @@ test("avoid generating collection parameter variable as state", () => {
 test("generate params variable when present", () => {
   expect(
     generatePageComponent({
+      classesMap: new Map([["body", ["cls"]]]),
```

Comment:
Write separate test to check classes. These cases checks unrelated stufff.

Reply:
agree

Reply:
done